Dockerfile.fedora

```diff
@@ -2,22 +2,19 @@ FROM fedora
 MAINTAINER langdon <langdon@fedoraproject.org>
 #Derived from Vaclav Pavlin <vpavlin@redhat.com>; https://github.com/vpavlin/atomicapp-run
 
-RUN yum -y update && yum clean all
-RUN yum -y install python python-pip docker && yum clean all
-
-ADD requirements.txt /opt/atomicapp/
-ADD setup.py /opt/atomicapp/
-ADD atomicapp/ /opt/atomicapp/atomicapp/
-
-WORKDIR /opt/atomicapp
-RUN python setup.py install
+RUN yum -y update
+RUN yum -y install python python-pip docker-io kubernetes && yum clean all
 
+COPY . /opt/atomicapp/
```

Comment:
I don't want to use `COPY .` because my workdir often contains a lot of random files and dirs created during development and I don't want to add them in the image.